release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -84,6 +84,13 @@ The {product-title} 4.10 installer uses new default component types for installa
 * AWS EC2 M6i instances for both control plane and compute nodes, where available
 * AWS EBS gp3 storage
 
+[id="ocp-4-10-enhancements-to-API-in-installconfig"]
+==== Enhancements to API in install.config
+
+Previously, when a user installed OCP on a Baremetal IPI platform, they had nowhere to configure custom network interfaces, such as static IPs or vLANs to communicate with the Ironic server.
```

Comment:
We don't use IPI in the docs, can you spell out "installer-provisioned infrastructure?

Reply:
Updated as suggested.